src/libs/actions/Timing.js

```diff
@@ -32,29 +32,32 @@ function end(eventName, secondaryName = '') {
         return;
     }
 
-    const {startTime, shouldUseFirebase} = timestampData[eventName];
-    const eventTime = Date.now() - startTime;
+    Environment.getEnvironment().then((envName) => {
+        const {startTime, shouldUseFirebase} = timestampData[eventName];
```

Comment:
Adding this inside a promise caused https://github.com/Expensify/App/issues/33290